packages/backend/src/app/authentication/authentication-service/hooks/default-authentication-service-hooks.ts

```diff
@@ -24,10 +23,9 @@ export const defaultAuthenticationServiceHooks: AuthenticationServiceHooks = {
             projectType: project.type,
         })
 
-        telemetry.identify(user, project.id)
-            .catch((e) => logger.error(e, '[AuthenticationService#signUp] telemetry.identify'))
+        await telemetry.identify(user, project.id)
```

Comment:
I always wanted to be async so it continue if this throws an error

Reply:
what about order? does `identify` need to come before `trackProject`?

Reply:
oh I see identify has to come first